packages/astro/src/types/public/config.ts

```diff
@@ -899,6 +899,22 @@ export interface AstroUserConfig {
 				defaultStrategy?: 'tap' | 'hover' | 'viewport' | 'load';
 		  };
 
+	/**
+	 * @docs
+	 * @name disableTelemetry
+	 * @type {boolean}
+	 * @version 5.0.0
+	 * @default `false`
+	 * @description
+	 * Disable anonymous telemetry collection for this project.
+	 *
+	 * You can also disable telemetry globally by running `astro telemetry disable`.
```

Comment:
```suggestion
	 * You can also disable telemetry globally from the CLI by running [`astro telemetry disable`](/en/reference/cli-reference/#astro-telemetry)`.
```
Just thought the link might be handy here! That section also has a tip for setting the environment variable, so might be nice to surface it.